scripts/known_good.json

```diff
@@ -8,6 +8,24 @@
             "install_dir": "Vulkan-Headers/build/install",
             "commit": "v1.3.243"
         },
+        {
+            "name": "Vulkan-Utility-Libraries",
+            "url": "https://github.com/KhronosGroup/Vulkan-Utility-Libraries.git",
+            "sub_dir": "Vulkan-Utility-Libraries",
+            "build_dir": "Vulkan-Utility-Libraries/build",
+            "install_dir": "Vulkan-Utility-Libraries/build/install",
+            "commit": "main",
+            "deps": [
+                {
+                    "var_name": "VULKAN_HEADERS_INSTALL_DIR",
+                    "repo_name": "Vulkan-Headers"
+                },
+                {
+                    "var_name": "GOOGLETEST_INSTALL_DIR",
+                    "repo_name": "googletest"
+                }
```

Comment:
Pass `-D UTILITY_LIBRARIES_BUILD_TESTS=OFF` instead.

This avoids having to deal with this dependency.

Reply:
Yes that's great, thanks!